web/src/app/rewardz/components/dashboard/rewards/rewards.component.ts

```diff
@@ -10,76 +10,101 @@ import {Token} from "../../../../common/data/token";
 import {Script} from "../../../../common/data/script";
 import {SpoRewardClaim} from "../../../../common/data/spo-reward-claim";
 import {WalletService} from "../../../../common/services/wallet.service";
+import {Title} from "@angular/platform-browser";
 
 @Component({
     selector: 'rewards',
     styleUrls: ['../../../styles/page-content.css'],
     templateUrl: './rewards.html'
 })
 
-export class RewardsComponent extends NotificationComponent implements OnInit {
+export class RewardsComponent extends NotificationComponent implements OnInit, OnDestroy {
     public walletSubscription: Subscription;
     public claimSubscription: Subscription;
     public tokens: Array<Token> = [];
-    public submitURL: string = "/rwd/multisig/6/sporwc";
-    public listURL: string = "/rwd/listURL";
-    public tokenCols: any[] = [];
+    public selectedTokens: Map<string, Token> = new Map<string, Token>();
     public claimReturn: string;
     public listReturn: string;
     public walletLoaded: boolean = false;
-    @ViewChild('fileUpload', {static: false}) public fileUpload: any;
+    public initialized: boolean = false;
+    public scanURLs = ["https://explorer.cardano-testnet.iohkdev.io/tx/", "https://explorer.cardano.org/tx/"];
```

Comment:
I personally like cardanoscan more it has more detailed overview but its just taste

Reply:
I can change it